distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -28,7 +28,7 @@ log4j.appender.out.maxBackupIndex=10
 # File appender - events.log
 log4j.appender.event=org.apache.log4j.RollingFileAppender
 log4j.appender.event.layout=org.apache.log4j.PatternLayout
-log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%-20.20c{1}] - %m%n
+log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%c{1}] - %m%n
```

Comment:
When I was testing it, there was so much white space and sometimes the name got truncated.
Then I tried it without restriction and in my opinion, it works out rather nice and even adds some readability because there are so little different names being used. This way, it becomes easy to scan the log for specific types of events.


```
2016-01-12 10:53:02.140 [GroupItemStateChangedEvent] - FF_Bath changed from NULL to UNDEF through Light_FF_Bath_Ceiling
2016-01-12 10:53:02.141 [ItemStateChangedEvent] - Light_FF_Bath_Ceiling changed from NULL to ON
2016-01-12 10:53:02.142 [ItemStateChangedEvent] - Light_FF_Bath_Mirror changed from NULL to OFF
2016-01-12 10:53:02.143 [GroupItemStateChangedEvent] - FF_Corridor changed from NULL to OFF through Light_FF_Corridor_Ceiling
2016-01-12 10:53:02.144 [ItemStateChangedEvent] - Light_FF_Corridor_Ceiling changed from NULL to OFF
2016-01-12 10:53:02.145 [GroupItemStateChangedEvent] - FF_Office changed from NULL to UNDEF through Light_FF_Office_Ceiling
2016-01-12 10:53:02.145 [ItemStateChangedEvent] - Light_FF_Office_Ceiling changed from NULL to OFF
2016-01-12 10:53:02.146 [GroupItemStateChangedEvent] - FF_Child changed from NULL to UNDEF through Light_FF_Child_Ceiling
2016-01-12 10:53:02.147 [ItemStateChangedEvent] - Light_FF_Child_Ceiling changed from NULL to OFF
2016-01-12 10:53:02.147 [GroupItemStateChangedEvent] - FF_Bed changed from NULL to UNDEF through Light_FF_Bed_Ceiling
2016-01-12 10:53:02.148 [ItemStateChangedEvent] - Light_FF_Bed_Ceiling changed from NULL to ON
2016-01-12 10:53:02.149 [GroupItemStateChangedEvent] - gC changed from NULL to UNDEF through Light_C_Corridor_Ceiling
2016-01-12 10:53:02.149 [ItemStateChangedEvent] - Light_C_Corridor_Ceiling changed from NULL to OFF
2016-01-12 10:53:02.150 [ItemStateChangedEvent] - Light_C_Staircase changed from NULL to OFF
2016-01-12 10:53:02.151 [ItemStateChangedEvent] - Light_C_Washing_Ceiling changed from NULL to OFF
2016-01-12 10:53:02.152 [GroupItemStateChangedEvent] - gC changed from UNDEF to OFF through Light_C_Workshop
2016-01-12 10:53:02.152 [ItemStateChangedEvent] - Light_C_Workshop changed from NULL to OFF
2016-01-12 10:53:02.153 [GroupItemStateChangedEvent] - Garden changed from NULL to UNDEF through Light_Garden_Garage
2016-01-12 10:53:02.153 [ItemStateChangedEvent] - Light_Garden_Garage changed from NULL to OFF
2016-01-12 10:53:02.154 [ItemStateChangedEvent] - Light_Garden_Terrace changed from NULL to OFF
2016-01-12 10:53:02.155 [GroupItemStateChangedEvent] - Heating changed from NULL to OFF through Heating_GF_Corridor
2016-01-12 10:53:02.155 [ItemStateChangedEvent] - Heating_GF_Corridor changed from NULL to OFF
2016-01-12 10:53:02.156 [ItemStateChangedEvent] - Heating_GF_Toilet changed from NULL to OFF
2016-01-12 10:53:02.157 [ItemStateChangedEvent] - Heating_GF_Living changed from NULL to OFF
2016-01-12 10:53:02.157 [ItemStateChangedEvent] - Heating_GF_Kitchen changed from NULL to OFF
```

But in the end, it's really about personal taste. If you want, I can change it back to the 20.